packages/cli/src/commands/integration/command.ts

```diff
@@ -34,4 +19,62 @@ export const integrationCommand: Command = {
       ],
     },
   ],
-};
+} as const;
+
+export const listSubcommand = {
+  name: 'list',
+  description: 'Lists all resources from marketplace integrations',
+  arguments: [],
+  options: [
+    {
+      name: 'integration',
+      description: 'limits the resources listed to a designated integration',
+      shorthand: 'i',
+      type: String,
+      deprecated: false,
+      argument: 'NAME',
+    },
+    {
+      name: 'current-project',
+      description: 'limits the resources listed to the current project',
+      shorthand: 'cp',
```

Comment:
shorthand flags must be one character

Reply:
Good to know, updated it to just `c`, thanks! 🙂